src/modals/Modal.tsx

```diff
@@ -30,6 +32,8 @@ type ModalProps = {
     closable?: boolean;
     onClose?: Function;
     dataqa?: string;
+    width?: number;
```

Comment:
@maheshkulkarni4 along with this can you please add `className `and style `props `? Those will be useful in future as well.

Reply:
Added a new Prop for ClassName.  As discussed with @akate1 , we are not adding prop for custom inline styling. Let developers use the custom classes to achieve same.